e2e-tests/playwright/e2e/plugins/quick-access-and-tech-radar.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { test, expect } from '@playwright/test';
+import { HomePage } from '../../support/pages/HomePage';
+import { Common } from '../../utils/Common';
+import { UIhelper } from '../../utils/UIhelper';
+import { TechRadar } from '../../support/pages/TechRadar';
+
+// Pre-req: Enable backstage-plugin-tech-radar Plugin
+
+test.describe('Test Customized Quick Access and tech-radar plugin', () => {
+  test.beforeEach(async ({ page }) => {
```

Comment:
Can we update it to the `beforeAll` hook?

Reply:
@subhashkhileri  if we change it to `beforeAll` we will receive this error:
```
Error: "context" and "page" fixtures are not supported in "beforeAll" since they are created on a per-test basis.
If you would like to reuse a single page between tests, create context manually with browser.newContext(). See https://aka.ms/playwright/reuse-page for details.
If you would like to configure your page before each test, do that in beforeEach hook instead
```
Let's create a specific Jira to create shared pages to use it on `beforeAll`